lerna.json

```diff
@@ -1,7 +1,7 @@
 {
-	"lerna": "2.0.0",
-	"packages": [
-		"packages/*"
-	],
-	"version": "independent"
+  "lerna": "2.8.0",
```

Comment:
They use 2.0.0 in the official docs, it seems strange to have to update the version in both places. Shouldn't we reach out Lerna team and clarify that?

Reply:
Nops! They recommend 2.x.x version.

Reply:
How did you initialize the lerna repo? You should run something like

Reply:
Actually lerna config was already here, that's why I had no idea how it should work :)

Reply:
Glad I could help. Let's merge this one :) 👋